### PR TITLE
Fix Class Paths (v1)

### LINKED ISSUE #10: Emarsys::Contact should be located in app/services/workarea
**Describe the bug**
Emarsys::Contact is not in app/services/workarea folder, therefore when I try to decorate I get a `NameError: uninitialized constant Workarea::Emarsys::Contact`

**To Reproduce**
1. Try to decorate Emarsys::Contact
2. Start up app

**Expected behavior**
App starts up and uses my decorated method

**Workarea Setup:**
 ```
  workarea (= 3.3.35)!
  workarea-ab_testing (= 2.1.0)!
  workarea-afterpay (= 2.1.0)!
  workarea-api (= 4.3.1)!
  workarea-basic_auth (~> 1.0.1)!
  workarea-browse_option (= 2.0.0)!
  workarea-clothing (= 2.1.10)!
  workarea-commerce_cloud (= 1.1.0)!
  workarea-drawer (= 0.1.2)!
  workarea-emarsys (= 1.0.3)!
  workarea-filter_dropdowns (= 3.0.2)!
  workarea-gift_cards (= 3.4.7)!
  workarea-global_e (= 1.1.0)!
  workarea-google_address_autocomplete (= 4.0.1)!
  workarea-google_product_feed (= 3.0.2)!
  workarea-google_tag_manager (= 4.2.6)!
  workarea-hearts (= 3.0.3)!
  workarea-hover_zoom (= 2.0.0)!
  workarea-import (= 3.0.5)!
  workarea-new_relic (= 1.0.1)!
  workarea-oms (= 3.0.0)!
  workarea-paypal (= 2.0.9)!
  workarea-product_grid_content (= 1.2.0)!
  workarea-segmentation (= 2.3.3)!
  workarea-sentry (~> 1.1.1)!
  workarea-share (= 1.2.4)!
  workarea-sitemaps (= 2.1.6)!
  workarea-slick_slider (= 0.1.0)!
  workarea-store_locator (= 4.1.0)!
  workarea-stripe (= 1.0.0.beta.3)!
  workarea-super_hero (= 2.0.5)!
  workarea-upgrade (~> 2.0.7)!
```

### PR DESCRIPTION
**Pull request by tubbo**
Some paths in `app/` did not match their constant names, resulting in
issues when trying to decorate these classes. Move the offending files
to their correct location so as to enable decorating them.

(I forgot to merge this into `v1.0-stable`)

Fixes #10

### CODE REVIEW

**Comment by tubbo** [comment]
The only difference in this one is that I moved `Workarea::Admin::Emarsys::ConfigurationsController` to its proper file. This class is no longer available in v2, but it still exists in v1 so we should probably make sure it has the right path.

File: app/assets/javascripts/workarea/storefront/emarsys/modules/web_extend_adapter.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ WORKAREA.analytics.registerAdapter('webExtendAdapter', function () {
 
     var pushEmarsysEvent = function(event, payload) {
             if (WORKAREA.environment.isTest || _.isUndefined(window.ScarabQueue)) {
-                console.log('Emarsys is not configured or is disabled in test');
+                if (!WORKAREA.environment.isTest) {
+                    console.log('Emarsys is not configured');
```
Comment (bencrouse) [inline app/assets/javascripts/workarea/storefront/emarsys/modules/web_extend_adapter.js]: You could adjust the above conditional to
```
if (!WORKAREA.environment.isTest && _.isUndefined(window.ScarabQueue) {
  console.log('Emarsys is not configured');
}
```

**Review by bencrouse — CHANGES_REQUESTED** [review]
Looks like rubocop is failing :(

File: app/assets/javascripts/workarea/storefront/emarsys/modules/web_extend_adapter.js (hunk 1 above)
Comment (tubbo) [reply app/assets/javascripts/workarea/storefront/emarsys/modules/web_extend_adapter.js]: this is totally out of scope for this change so i'm removing it. we can address the issue with JS in the tests in a different PR.

Comment (tubbo) [reply app/assets/javascripts/workarea/storefront/emarsys/modules/web_extend_adapter.js]: turns out this was masking the actual failures which I fixed

**Review by bencrouse — APPROVED** [review]

**Pull request merged**